.ci/openshift_e2e.sh

```diff
@@ -30,26 +30,26 @@ function Catch_Finish() {
 export CI="openshift"
 export ARTIFACTS_DIR="/tmp/artifacts"
 export NAMESPACE="devworkspace-controller"
-
 # Component is defined in Openshift CI job configuration. See: https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v4.yaml#L8
 export CI_COMPONENT="devworkspace-operator"
-
 # DEVWORKSPACE_OPERATOR env var exposed by Openshift CI in e2e test pod. More info about how images are builded in Openshift CI: https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
 # Dependencies environment are defined here: https://github.com/openshift/release/blob/master/ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-master__v5.yaml#L36-L38
+
 export IMG=${DEVWORKSPACE_OPERATOR}
 
 # Pod created by openshift ci don't have user. Using this envs should avoid errors with git user.
 export GIT_COMMITTER_NAME="CI BOT"
 export GIT_COMMITTER_EMAIL="ci_bot@notused.com"
 
-# Function to get all logs and events from devworkspace operator deployments
+# For some reason go on PROW force usage vendor folder
+# This workaround is here until we don't figure out cause
```

Comment:
I think this comment still needs to be moved back down to where the `go mod tidy` and `go mod vendor` are